--- maxr-0.2.1/Makefile.am~	2008-12-23 09:41:26.000000000 +0100
+++ maxr-0.2.1/Makefile.am	2008-12-23 09:41:11.000000000 +0100
@@ -6,8 +6,8 @@
 
 # assign parameters from configure
 prefix=@prefix@
-#bindir=@prefix@/maxr
-DATADIR=@prefix@/maxr
+BINDIR=@bindir@
+DATADIR=@datadir@/maxr
 INSTALL=$(top_srcdir)/install-sh
 
 
@@ -15,7 +15,8 @@
 #	$(INSTALL) -d $(DATADIR)/data
 #	$(INSTALL) -d $(DATADIR)/2
 # Copy each compiled file to datadir (with path)
-	$(INSTALL) -d $(DATADIR)
-	cp -a $(top_srcdir)/data/* $(DATADIR)/
-	$(INSTALL) -m 755 $(top_srcdir)/src/maxr $(DATADIR)/
+	$(INSTALL) -d $(DESTDIR)$(DATADIR)
+	cp -a $(top_srcdir)/data/* $(DESTDIR)$(DATADIR)/
+	$(INSTALL) -d $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 $(top_srcdir)/src/maxr $(DESTDIR)$(BINDIR)/
 
